src/Aspire.Dashboard/Components/Pages/Metrics.razor

```diff
@@ -107,10 +111,12 @@
                                         }
                                         else
                                         {
-                                            // class needed to prevent color from being removed from reboot style
-                                            <a class="" @onclick="@(async () => await _contentLayout!.OpenMobileToolbarAsync())">
-                                                @Loc[nameof(Dashboard.Resources.Metrics.MetricsSelectInstrument)]
-                                            </a>
+                                            <div class="empty-content">
+                                                <!-- class needed to prevent color from being removed from reboot style -->
+                                                <a class="" @onclick="@(async () => await _contentLayout!.OpenMobileToolbarAsync())">
```

Comment:
Use FluentAnchor

See https://www.fluentui-blazor.net/Anchor and the `Link with no Href` example.